nipype/algorithms/tests/test_confounds.py

```diff
@@ -36,6 +37,21 @@ def test_fd(tmpdir):
         ground_truth, np.loadtxt(res.outputs.out_file, skiprows=1), atol=.16)
     assert np.abs(ground_truth.mean() - res.outputs.fd_average) < 1e-2
 
+    fdisplacement = FramewiseDisplacement(
+        in_file=example_data('fsl_mcflirt_movpar.txt'),
+        out_file=tempdir + '/fd.txt',
+        parameter_source="FSL",
+        metric='riemannian')
+    res = fdisplacement.run()
+
+    with open(res.outputs.out_file) as all_lines:
+        for line in all_lines:
+            assert 'FramewiseDisplacement' in line
+            break
+
+    assert np.allclose(
+        ground_truth, np.loadtxt(res.outputs.out_file, skiprows=1), atol=.16)
```

Comment:
Going to guess we're going to need a different `ground_truth` file, but that's probably going to require figuring out the Euler angles first. So this is just a flag so we don't forget about it.